scss/_patterns_lists.scss

```diff
@@ -133,11 +119,21 @@ $list-status-icon-nudge: 0.3rem; // 0.3rem seems to be a magic number that posit
     .p-list__item {
       @extend %vf-list-item;
       @include vf-list-item-divided;
+
+      &.is-ticked {
+        background-position-y: 0.85rem;
```

Comment:
Could this be expressed as variables as well?

Not sure what this value is, but if it's based of icon size it should be calculated from it.

Reply:
This can go away! I just forgot to remove it